map/src/data/missingPlanets.json

```diff
@@ -115,15 +115,15 @@
         "grid": "P4",
         "link": "",
         "name_web": "",
-        "point_y": 8654.09045621,
-        "point_x": 6373.41681693,
-        "y": 8654.09,
-        "x": 6373.42
+        "point_y": 7700.09045621,
+        "point_x": 6855.41681693,
+        "y": 7700.09,
+        "x": 6855.42
       }
     },
     {
       "type": "Feature",
-      "geometry": { "type": "Point", "coordinates": [57.253377, 61.122869] },
+      "geometry": { "type": "Point", "coordinates": [60.953377, 55.122869] },
```

Comment:
I need to tweak these slightly, this knocks Carshoulis down into square P5 instead of being in P4.